ra-optimisation/rao-commons/src/main/java/com/farao_community/farao/rao_commons/linear_optimisation/FaraoMPSolver.java

```diff
@@ -7,37 +7,86 @@
 
 package com.farao_community.farao.rao_commons.linear_optimisation;
 
-import com.google.ortools.linearsolver.MPConstraint;
-import com.google.ortools.linearsolver.MPSolver;
-import com.google.ortools.linearsolver.MPVariable;
-import com.google.ortools.linearsolver.mainJNI;
+import com.farao_community.farao.rao_commons.RaoUtil;
+import com.google.ortools.linearsolver.*;
+
+import java.util.HashMap;
+import java.util.Map;
 
 /**
  * @author Philippe Edwards {@literal <philippe.edwards at rte-international.com>}
  */
 public class FaraoMPSolver extends MPSolver {
 
-    private static final double PRECISION = 1048576; //2^20
+    private static final double PRECISION = 1073741824; //2^30
```

Comment:
not easy to read. Could we have something like NUMBER_OF_TRUNCATED_BITS_FOR_DOUBLE_PRECISION (=30), or something more explicit ?